models/Discord/Commands/DiscordCommandWhoIs.js

```diff
@@ -33,9 +33,9 @@ class DiscordCommandWhoIs extends DiscordCommand {
       var userID = auser
       
       if(userID) {
-        getByDiscordID(connection, userID, message)
+        getByDiscordID(connection, userID, message, dbSubsystem.format_table_name('player'))
       } else {
-        getByCkey(connection, provided_ckey, message)
+        getByCkey(connection, provided_ckey, message, dbSubsystem.format_table_name('player'))
```

Comment:
WTF is this, set it in the query

Reply:
The function doesn't have access to dbSubsystem

Reply:
Then make it have it. Move the function in scope.